activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb

```diff
@@ -53,8 +53,10 @@ def initialize(connection, logger, connection_options, config)
 
         @statements = StatementPool.new(self.class.type_cast_config_to_integer(config[:statement_limit]))
 
-        if version < "5.1.10"
-          raise "Your version of MySQL (#{version_string}) is too old. Active Record supports MySQL >= 5.1.10."
+        if version < "5.7.9"
+          raise "Your version of MySQL (#{version_string}) is too old. Active Record supports MySQL >= 5.7.9."
+        elsif (mariadb? && version < "5.6.33")
```

Comment:
What does 5.6.33 mean?

Reply:
When I tested at Travis CI by setting too high version
to show MariaDB 10.3 version. It was "5.6.33". 

```
/home/travis/build/rails/rails/activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb:57:in `initialize': Your version of MySQL (5.6.33) is too old. Active Record supports MySQL >= 5.7.9. (RuntimeError)
```

But it must be wrong there may be something wrong since I installed MariaDB 10.3 locally and it says "10.3.9" as follows.

```
Your version of MariaDB (10.3.9) is too old. Active Record supports MariaDB >= 10.2.2 (RuntimeError)
```